packages/core/src/package/scriptExecutors/scriptExecutorHelpers.ts

```diff
@@ -0,0 +1,19 @@
+import ExecuteCommand from '../../command/commandExecutor/ExecuteCommand';
+import SFPLogger, { Logger, LoggerLevel } from '../../logger/SFPLogger';
+import defaultShell from '../../utils/DefaultShell';
+
+export default class scriptExecutorHelpers {
```

Comment:
Can you rename this to start with Caps.. to align with other Files

Reply:
Also lets probably move this from package section, as this is now used elsewhere